src/v2/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -174,10 +174,10 @@ export class ArtworkGridContainer extends React.Component<
     const nodes = extractNodes(artworks)
 
     return (
-      <GridColumns>
-        {nodes.map((artwork, index) => {
+      <GridColumns width="100%">
```

Comment:
As far as layout being compressed; this was your only problem; since it was a flex-child.